packages/core/rendering/core-rendering-server-internal/src/rendering_service.tsx

```diff
@@ -91,6 +91,10 @@ export class RenderingService {
       mode: this.coreContext.env.mode,
       packageInfo: this.coreContext.env.packageInfo,
     };
+    const config = this.coreContext.configService.getConfig$();
+    // @ts-expect-error
+    const { whitelabelling } = (await config.pipe(take(1)).toPromise())?.rawConfig;
+    const theming = whitelabelling?.theme;
```

Comment:
This part will have to change unfortunately. We can't have Core blindly access some random plugin's raw configuration. We will need an explicit (setup-phase) API for the `whitelabelling` plugin to register these theming info in a similar way to what you did on the browser side. Having a server-side `theme` service to do so seems like the correct approach too (the rendering service would then access the theme / whitelabelling info from the theme service)